test/dynamo/test_dynamo.py

```diff
@@ -502,7 +502,9 @@ def test_resnet18(self):
     # Graph 3: optimizer
     # Graph 4: sync input for backward
     # Graph 5: sync input for backward (TODO(JackCaoG) understand why there are two graphs)
-    self.assertEqual(met.metric_data('CompileTime')[0], 5)
+    # Graph 6, 7: PyTorch has updated the number of captured by resnet with
+    # https://github.com/pytorch/pytorch/pull/117434. 
+    self.assertEqual(met.metric_data('CompileTime')[0], 7)
```

Comment:
do we know why we compile 2 more graphs?

Reply:
Yes because previously SGD was disabled, now it's enabled :)

Reply:
oh ok, that make sense!